conda_forge_tick/auto_tick.xsh

```diff
@@ -451,6 +451,7 @@ def initialize_migrators(do_rebuild=False):
     add_rebuild_successors($MIGRATORS, gx, 'proj4', '6.1.0')
     add_rebuild_successors($MIGRATORS, gx, 'pyqt', '5.9.2')
     add_rebuild_successors($MIGRATORS, gx, 'glog', '0.4.0')
+    add_rebuild_successors($MIGRATORS, gx, 'boost-cpp', '1.70.0')
```

Comment:
I'm assuming it will rebuild successors of successors, so a rebuild-successors for just `'boost'` won't be necessary (since `boost` is a `boost-cpp` successor), but I didn't verify this

Reply:
Only direct successors will be rebuilt

Reply:
That was a fast answer :) 
I'll add `boost`, thanks